src/common/model_utils/tests/test_preprocessing_blacklist.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+import sys
+import os
+import pytest
+
+CWD = Path(__file__).resolve()
+sys.path.append(str(CWD.parents[2]))  # common/ dir
+
+from model_utils.preprocessing import filter_blacklisted_qrcodes  # noqa: E402
+
+
+BASE_PATH = "this/is/the/test/path"
+
+
+def prepare_qrs():
+    QRCODES_11SAMPLES = [
```

Comment:
```suggestion
    qrcodes_samples = [
```

Please also make other local variables below lowercase, as this is our convention